go/tools/gopackagesdriver/bazel.go

```diff
@@ -36,6 +36,7 @@ const (
 type Bazel struct {
 	bazelBin      string
 	workspaceRoot string
+	bazelFlags    []string
```

Comment:
Nit: following the convention of [bazelrc](https://bazel.build/run/bazelrc), it's more clear to call them `startupFlags` or `startupOptions`.